src/doc/rustc/src/platform-support/m68k-unknown-none-elf.md

```diff
@@ -6,7 +6,7 @@ Bare metal Motorola 680x0
 
 ## Designated Developers
```

Comment:
Aside: why do some targets say "designated developers" and others "target maintainers"?

Reply:
They are the same thing really, can just rename to Target maintainers